#################################################
#  SLAM/GPS Pose graph optimization parameters  #
#################################################

output_frame_id: "gps_init"

n_iterations: 1000
time_offset: 0.  # GPS time = SLAM time + TimeOffset
tf_sensor_to_gps: [0.121, -0.197, 0.173, 0., -0.3, 0.]  # Pose (x, y, z, rx, ry, rz) of the GPS in the referential of the sensor that produce the trajectory to optimize
verbose: true
g2o_file_name: "/home/nicolas/pose_graph_optimization.g2o"